README.md

```diff
@@ -1,140 +1,32 @@
-
-Fluent UI React is a collection of robust React-based components designed to make it simple for you to create consistent web experiences using the Fluent Design Language.
-
-## Learn about Fluent UI
-
-[Fluent UI React current release documentation](https://developer.microsoft.com/en-us/fluentui)
-
-[Fluent UI Northstar documentation for Teams developers](https://aka.ms/fluent-ui)
-
-## Who uses Fluent UI React?
-
-![image](https://user-images.githubusercontent.com/785361/50458071-45b58d00-0915-11e9-90c0-ad8789c99db5.png)
-
-\+ 45 additional Microsoft sites and products
-
-## For more information...
-
-Please see the [wiki](https://github.com/microsoft/fluentui/wiki).
-
-## Contents
-
-- [Using Fluent UI React](#using-fluent-ui-react)
-  - [Integrating in your project](#integrating-in-your-project)
-  - [Version policy](#version-policy)
-  - [Browser support](#browser-support)
-  - [Right-to-left support](#right-to-left-support)
-  - [Server-side rendering](#server-side-rendering)
-  - [Advanced usage](#advanced-usage)
-- [Contribute to Fluent UI React](#contribute-to-fluent-ui-react)
-- [Building the repo](#building-the-repo)
-  - [Testing](#testing)
-  - [Advanced building tips](#advanced-building-tips)
-- [Licenses](#licenses)
-- [Changelog](#changelog)
-
-## Using Fluent UI React
-
-### Creating a new app
-
-To create a simple React app using [Create React App](https://create-react-app.dev), [install Node.js](https://nodejs.org), then run:
-
-```sh
-npx create-react-app my-app
-cd my-app
-npm install @fluentui/react
-npm start
-```
-
-See the next section for some starter code using Fluent UI React controls.
-
-### Integrating in your project
-
-```sh
-npm i @fluentui/react
-
-# Or, use yarn
-yarn add @fluentui/react
-```
-
-This will add the package as a dependency in your `package.json` file and download it under `node_modules/@fluentui/react`.
-
-The library includes ES2015 module entry points under the `lib` folder (use `lib-amd` if you need AMD, or `lib-commonjs` if you need commonjs). To use a control, import it and then use it in your render method:
-
-```jsx
-import React from 'react';
-import ReactDOM from 'react-dom';
-import { PrimaryButton } from '@fluentui/react/lib/Button';
+Fluent UI web represents a collection of utilities, React components, and web components for building web applications.
 
-ReactDOM.render(<PrimaryButton>I am a button.</PrimaryButton>, document.getElementById('root'));
-```
+The Fluent UI web repository is home to 3 main projects today, the following table will help navigate them and understand their differences.
 
-### Version policy
+<!-- prettier-ignore-start -->
+|   | React | React Northstar | Web Components |
+|---| ----- | --------------- | -------------- |
+| **Overview**    | Mature, refreshing with new concepts from react-northstar. | Newer, has concepts we're iterating on. | Web Component implementation of Fluent UI. |
+| **Used By**     | Office| Teams | TODO |
```

Comment:
@chrisdholt same comment as prior PR. Can you help provide the "Used By" for Web Components?

Reply:
Let's go with "Edge" for now

Reply:
Cool, changed to Edge